tests/test_polytracker.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import os
+from polyprocess import PolyProcess
+
+cwd: str = os.getcwd()
```

Comment:
You could replace these paths with something like this:
```python
TEST_DIR = os.path.realpath(os.path.dirname(__file__))
BIN_DIR = os.path.join(TEST_DIR, 'bin')
TEST_RESULTS_DIR = os.path.join(BIN_DIR, 'test_results')
BITCODE_DIR = os.path.join(test_dir, 'bitcode')
```

Reply:
That sounds like a good idea, ill do that

Reply:
The first line will give you the absolute path to the directory in which the current script is located, regardless of the current working directory. This lets you call the code from a different CWD. Using `os.path.join` also makes the code portable, because it will handle Windows paths.